src/utils/storageFacade.utils.ts

```diff
@@ -324,14 +530,6 @@ async function createJunctionEntry(
   return status;
 }
 
-async function DeleteVuln(cveid: number) {
```

Comment:
why did you delete this function? would be helpful with duplicates and purging at the end of a session. Should it be in export - as in - after you export, your data is purged?